pkgs/build-support/setup-hooks/fix-darwin-dylib-names.sh

```diff
@@ -23,10 +23,13 @@ fixDarwinDylibNames() {
     for fn in "$@"; do
         if [ -L "$fn" ]; then continue; fi
         echo "$fn: fixing dylib"
+        set +e
         int_out=$(@targetPrefix@install_name_tool -id "$fn" "${flags[@]}" "$fn" 2>&1)
         result=$?
+        set -e
         if [ "$result" -ne 0 ] &&
-            ! grep "shared library stub file and can't be changed" <<< "$out"
```

Comment:
Any idea why `$out` was being grepped before? Was that a mistake or is this a string that occurs in stubs rather than in `install_name_tool` output?

Reply:
It is a mistake, as also addressed in #267213. Alhough without `set +e` the build will stop at the `install_name_tool` line on failure and won't even reach the grep here.